docs/sources/mimir/query/query-metric-labels.md

```diff
@@ -0,0 +1,117 @@
+
+### Features comparison
+
+The different API endpoints have different features, and the main differences follow.
+
+#### Can specify the time range in the request?
+
+- **Get label names**<br />
+  Yes. However, the actual time range granularity is restricted to TSDB block range periods, which are `2h` for the most recent metrics, and up to `24h` for historical metrics when running Mimir with the default configuration.
+  In practice, the input `start` and `end` parameters are rounded to the TSDB block boundaries:
+  - The `start` parameter is rounded to the start of the nearest block containing samples with timestamps that are greater than or equal to the input timestamp.
+  - The `end` parameter is rounded to the end of the nearest block containing samples with timestamps that are less than or equal to the input timestamp.
+- **Label names cardinality**<br />
+  No. The API queries only the in-memory series in the ingesters that hold series for the tenant.
+
+#### Can specify the series selector in the request?
+
+- **Get label names API**<br />
+  Yes. The optional `match[]` parameter makes it possible to select the series from which to read the label names. The `match[]` parameter can be specified multiple times. When omitted, label names are extracted from all series.
+- **Label names cardinality API**<br />
+  Yes. The optional `selector` parameter allows to select the series from which to read the label names. The `selector` parameter can be specified only once. When omitted, label names are extracted from all series.
+
+## Querying label values
+
+The following API endpoints find the list of label values for a given label name:
+
+- **[Get label values]({{< relref "../references/http-api/index.md#get-label-values" >}})**<br />
+  `GET <prometheus-http-prefix>/api/v1/label/{name}/values`
+- **[Label values cardinality]({{< relref "../references/http-api/index.md#label-values-cardinality" >}})**<br />
+  `GET,POST <prometheus-http-prefix>/api/v1/cardinality/label_values`
+
+{{% admonition type="note" %}}
+`<prometheus-http-prefix>` is the Prometheus HTTP prefix as documented in the [HTTP API]({{< relref "../references/http-api/index.md#path-prefixes" >}}) reference.
+The default prefix is `/prometheus`.
+{{% /admonition %}}
+
+### Recommendations and performance trade-off
+
+- Prefer _get label values_ over _label values cardinality_ API, unless you need to know the number of series for each label name and value.
+- The _get label values_ API allows to specify a single label name, while the _label values cardinality_ API allows to specify multiple label names. If you need to get the label values for multiple label names, we recommend issuing multiple _get label values_ requests instead of a single _label values cardinality_ API call with multiple label names.
```

Comment:
```suggestion
- The _get label values_ API allows to specify a single label name, and the _label values cardinality_ API allows to specify multiple label names. If you need to get the label values for multiple label names, we recommend issuing multiple _get label values_ requests instead of a single _label values cardinality_ API call with multiple label names.
```

Reply:
"while¨ means "and" in this context.